tests/unit/synapseutils/unit_test_synapseutils_migrate.py

```diff
@@ -1838,13 +1899,58 @@ def test__verify_index_settings__retrieve_index_settings():
                 cursor,
                 db_path,
                 root_id,
-                storage_location_id,
+                dest_storage_location_id,
+                source_storage_location_ids,
+                'changed',
+                include_table_files
+            )
+        assert 'changed' in str(ex.value)
```

Comment:
assert more of the error message. maybe refactor error message such that the variable related strings are near the end of the message.